script.js

```diff
@@ -1,4 +1,452 @@
+  if (userRound == user.length) {
+    return normalModeResult(choice);
+  }
+  return `You choose ${choice} to place first , your number is ${
+    playerNumber[userRound - 1]
+  }.🥳🥳<br> Next player ${user[userRound]} please roll your dices!`;
+};
+
+var normalModeResult = function (dice) {
+  let result = `You choose ${dice} to place first , your number is ${
+    playerNumber[userRound - 1]
+  }.<br> Everyone have already roll their dices and choose their number.🎲🎲🎲<br>Here is the list of the player and the number <br>${genUserNumberList()}<br> Congrats${scorePointAndGenList(
+    true
+  )}🎉🎉. You Wins! Now the score is in below:<br>${genUserList()}`;
+  endGame();
+  return result;
+};
+
+//AFS Mode function
+var rollDiceAFSMode = function () {
+  let dice1 = genDice();
+  let dice2 = genDice();
+  let currentNumber = currentNumberAFS(dice1, dice2);
+
+  if (userRound == user.length - 1 && accumulatedRound == 4) {
+    return AFSModeResult(dice1, dice2, currentNumber);
+  }
+  if (userRound == user.length - 1 && accumulatedRound < 4) {
+    userRound = 0;
+    accumulatedRound += 1;
+    winnerList = scorePointAndGenList(false);
+    return `${user[user.length - 1]} roll ${dice1} and ${dice2}.🎲🎲<br> The ${
+      currentNumber[1]
+    } combination number is ${
+      currentNumber[0]
+    }.<br><br>This is the end the Round ${accumulatedRound}.😎<br>Let's see the result until now.<br><br>${genUserNumberList()}<br>It will be hard for you${winnerList}.<br>The next round will use the smallest combination for the current winner.<br>Good Luck.😏😏`;
+  }
+
+  let result = `${
+    user[userRound]
+  } have rolled ${dice1} and ${dice2}.🎲🎲<br> The ${
+    currentNumber[1]
+  } combination number is ${currentNumber[0]}.<br>Next player ${
+    user[userRound + 1]
+  } please roll your dices!🤩🤩`;
+  userRound += 1;
+  return result;
+};
+
+var currentNumberAFS = function (dice1, dice2) {
+  let currentNumber = 0;
+  let largeOrSmall = "largest";
+  let output = [];
+  if (dice1 >= dice2 && accumulatedRound == 0) {
+    currentNumber = Number(String(dice1) + String(dice2));
+    playerNumber.push(currentNumber);
+  } else if (dice1 < dice2 && accumulatedRound == 0) {
+    currentNumber = Number(String(dice2) + String(dice1));
+    playerNumber.push(currentNumber);
+  } else if (dice1 >= dice2 && currentWinner.includes(userRound)) {
+    currentNumber = Number(String(dice2) + String(dice1));
+    playerNumber[userRound] += currentNumber;
+    largeOrSmall = "smallest";
+  } else if (dice1 < dice2 && currentWinner.includes(userRound)) {
+    currentNumber = Number(String(dice1) + String(dice2));
+    playerNumber[userRound] += currentNumber;
+    largeOrSmall = "smallest";
+  } else if (dice1 >= dice2) {
+    currentNumber = Number(String(dice1) + String(dice2));
+    playerNumber[userRound] += currentNumber;
+  } else if (dice1 < dice2) {
+    currentNumber = Number(String(dice2) + String(dice1));
+    playerNumber[userRound] += currentNumber;
+  }
```

Comment:
This looks like a lot of repetition, I would probably suggest doing it like this:
```
  let currentNumber = Math.max(dice1, dice2) * 10 + Math.min(dice1, dice2);
  if (accumulatedRound == 0) {
    playerNumber.push(currentNumber);
  } else {
    if (currentWinner.includes(userRound)) {
      currentNumber = Math.min(dice1, dice2) * 10 + Math.max(dice1, dice2);
      largeOrSmall = "smallest";
    }
    playerNumber[userRound] += currentNumber;
  }
```
I think it should cover the same cases!